### DISCUSSION TITLE
swag init command generated empty paths in json and yaml file

### DISCUSSION THREAD

**Discussion by SamarSidharth** [swaggo/swag#1595] [General]
I am new to using swaggo, I tried to generate swagger doc for my gin project but for some reason my API operation tags are not getting processed and the path field is always empty. The general API information in the main.go is processed correctly, but the API endpoint related swagger data in files under a sub directory are not processed.
Is swaggo looking for any particular thing in the file before it proccess it? 

I have tried it with -d option also but still the result is same.

Any help is much appreciated. 
Thanks

**Comment by baranitharan2020** [comment]
I'm also facing the same issue. But the same code generated the "paths" in another system. Very Strange

**Comment by d-honeybadger** [comment]
In my case empty paths were due to `runtime.GOROOT()` producting an empty string (and, I believe, since swag ignores all paths with `runtime.GOROOT()` prefix, it ignored all of them).
The reason `runtime.GOROOT()` would be empty could be b/c of the way `swag` is built. In my case it was a nix package (through devbox), with this issue: https://github.com/NixOS/nixpkgs/issues/224701
